src/main/java/com/getindata/connectors/http/sink/HttpSink.java

```diff
@@ -0,0 +1,78 @@
+package com.getindata.connectors.http.sink;
+
+import org.apache.flink.connector.base.sink.AsyncSinkBase;
+import org.apache.flink.connector.base.sink.writer.BufferedRequestState;
+import org.apache.flink.connector.base.sink.writer.ElementConverter;
+import org.apache.flink.core.io.SimpleVersionedSerializer;
+import org.apache.flink.util.Preconditions;
+
+import java.io.IOException;
+import java.util.Collection;
+import java.util.Collections;
+
+public class HttpSink<InputT> extends AsyncSinkBase<InputT, HttpSinkRequestEntry> {
+  private final String endpointUrl;
+
+  protected HttpSink(
+      ElementConverter<InputT, HttpSinkRequestEntry> elementConverter,
+      int maxBatchSize,
+      int maxInFlightRequests,
+      int maxBufferedRequests,
+      long maxBatchSizeInBytes,
+      long maxTimeInBufferMS,
+      long maxRecordSizeInBytes,
+      String endpointUrl
```

Comment:
Do we know/understand what those arguments are used for? I know that those comes from AsyncSinkBase but still would be good to know what they actually change/mean.

A javadoc on the constructor would help here.